jumpcloud-ADMU/Powershell/Form.ps1

```diff
@@ -661,13 +661,32 @@ $bMigrateProfile.Add_Click( {
             } else {
                 Write-ToLog "ConnectKey is populated, JumpCloud agent will be installed"
             }
-            $testResult, $userID = Test-JumpCloudUsername -JumpCloudApiKey $tbJumpCloudAPIKey.Text -JumpCloudOrgID $lbl_orgId.Text -Username $tbJumpCloudUserName.Text -Prompt $true
+
+            $testResult, $JumpCloudUserId, $jcUsername, $JCSystemUsername = Test-JumpCloudUsername -JumpCloudApiKey $tbJumpCloudAPIKey.Text -JumpCloudOrgID $lbl_orgId.Text -Username $tbJumpCloudUserName.Text -Prompt $true
             if ($testResult) {
                 Write-ToLog "Matched $($tbJumpCloudUserName.Text) with user in the JumpCloud Console"
             } else {
                 Write-ToLog "$($tbJumpCloudUserName.Text) not found in the JumpCloud console"
                 return
             }
+
+
+            if ($JCSystemUsername) {
+                Write-ToLog "User $($JCSystemUsername) has a local account on this system"
```

Comment:
This is a confusing statement. Hopefully the `$JCSystemUsername` is not a local account on the system. If there was a local user account with the same username value as `$JCSystemUsername` migration would fail.

But that brings up a good point. 

What happens when we have a JumpCloud with username: `bob` and systemUsername: `bobSmith` that needs to migrate from domain account `AzureAD\bobSmith`. Let's say that system already has a local user account called `bobSmith` that was created long ago. Before migration occurs, we'd not pick up the fact that these accounts would conflict. Due to the fact that we prevent users from migrating AD accounts to local accounts where the account already exists, we should also do that here. 

To be completely clear this screenshot illustrates the issue at hand:
![Screen Shot 2023-05-25 at 12 05 48 PM](https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/54448601/f8eef6e6-f958-4c5c-a85b-3361df3fcc53)

The "Migrate Profile" button is greyed out due to the fact that we are attempting to migrate `AzureAD\ChetAtkins` to be `bob` and `bob` already exists on the system as a local user. 

If we identify a JumpCloud user with a `systemUsername` we should be validating that both their `username` and systemUsername` do not exist as local user accounts on the device.